src/main/java/mate/academy/bookstore/repository/book/specification/IsbnSpecificationProvider.java

```diff
@@ -1,13 +1,22 @@
 package mate.academy.bookstore.repository.book.specification;
 
+import mate.academy.bookstore.dto.BookSearchParametersDto;
 import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.repository.SpecificationProvider;
 import mate.academy.bookstore.repository.book.BookSearchParameter;
+import org.springframework.data.jpa.domain.Specification;
 import org.springframework.stereotype.Component;
 
 @Component
-public class IsbnSpecificationProvider extends AbstractSpecificationProvider<Book> {
+public class IsbnSpecificationProvider implements SpecificationProvider<Book> {
     @Override
     public String getKey() {
         return BookSearchParameter.ISBN.getKey();
     }
+
+    @Override
+    public Specification<Book> getSpecification(BookSearchParametersDto params) {
+        return (root, query, criteriaBuilder) -> criteriaBuilder.like(
+                root.get(getKey()), "%" + params.isbn() + "%");
```

Comment:
we could actually organize a search by 2 fields, like so:
some `searchString` param that we get in dto, is `LIKE ISBN OR LIKE title` but that is a feature for another day 😄